selenium/che-selenium-test/src/test/resources/conf/selenium.properties

```diff
@@ -24,3 +24,21 @@ github.auxiliary.password=
 # Google credentials
 google.user=
 google.password=
+
+# Define configuration for che default test user.
+# Note that for single user assembly the configuration should be:
+# email: che@eclipse.org
+# password: secret
+# For che multiuser assembly, user must be already created on identity provider service
+che.test_user.email=
+che.test_user.password=
+
+# Defines additional test user
+che.aditional_test_user.email=
```

Comment:
I can't find the usage of this properties. Can you, please, write an example of usage?

Reply:
I'm working on it and will notify when it will be done

Reply:
@dmytro-ndp  I deleted these properties because for the first step of integration selenium tests with  che multiuser it's overhead.

Reply:
I see. We can add it in future on demand.